docs/src/using-turing/get-started.md

```diff
@@ -12,31 +12,23 @@ To use Turing, you need to install Julia first and then install Turing.
 
 ### Install Julia
 
-You will need to install Julia 1.0 or greater, which you can get from [the official Julia website](http://julialang.org/downloads/).
+You will need to install Julia 1.3 or greater, which you can get from [the official Julia website](http://julialang.org/downloads/).
```

Comment:
@cpfiffer let's warn the user about this breaking change.

Reply:
I'll add it to the Discourse post.